<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExposeConfirmationDialog</class>
 <widget class="QDialog" name="ExposeConfirmationDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Внимание!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkShowStats">
     <property name="text">
      <string>Выводить статистику</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkMesCheck">
     <property name="text">
      <string>Проверять на соответствие МЭС</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkReExpose">
     <property name="text">
      <string>Выполнять перевыставление по имеющимся отказам</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkCalendarDaysLength">
     <property name="text">
      <string>Длительность лечения в круглосуточном стационаре определяется в календарных днях</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblMessage">
     <property name="text">
      <string>message</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="chkLock">
     <property name="text">
      <string>Блокировать обращения в выставленных счетах</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="chkFilterPaymentByOrgStructure">
     <property name="text">
      <string>При выставлении счетов учитывать текущее подразделение</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkSeparateReExpose">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Перевыставлять в отдельный счет</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkAcceptNewKSLPForChild">
     <property name="text">
      <string>Применить КСЛП=1,8 для детей до 1 года (только для специализированных инфекционных стационаров краевого уровня)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkFilterPaymentByOrgStructure</tabstop>
  <tabstop>chkReExpose</tabstop>
  <tabstop>chkSeparateReExpose</tabstop>
  <tabstop>chkMesCheck</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExposeConfirmationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExposeConfirmationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkReExpose</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkSeparateReExpose</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
